schemaregistry/schemaregistry.md

```diff
@@ -404,4 +496,195 @@ The returned payload is the schema document. Further attributes such as the
+and consumers, producers will generally be the controlling authorities and the
+replication of schemas flows from registries near the producers to registries
+near the consumers.
+
+When schema governance is handled centrally, the replication of schemas might
+flow from a central registry to registries near the producers to registries
+near the consumers.
+
+These two models mights also be mixed. Schemas representing event data that is
+used throughout a greater system might stem from a centrally managed registry
+and event data schemas that are local to a subsystem might be controlled by the
+respective producers. 
+
+### 4.2 Replication topologies
+
+A peer-to-peer replication topology copies schemas between a source and target
+registry directly. For a mutual synchronization of schemas between registries, a
+second replication path is established with reversed roles.
+
+A hub replication topology copies schemas from a source to the target through an
+intermediary hub. The hub acts as an intermediary store, but with the authority
+over the schemas still being with the producers. Hubs might also be used by
+services that cannot host their own service registry endpoints.
+
+A star replication topology uses hubs central hubs into which all sources
+contribute schemas and to which all targets subscribe. If the hub is a centrally
+governed registry, it might not accept inbound replications.
+
+### 4.3 Push replication
+
+In an imperative "push" replication model, a source might push any updates of its
+state to one of more configured targets by forwarding all changes to the
+respective schema registry API calls on the target.
+
+The pushing source might be configured to copy changes of the full registry, of
+a specific schema group, or just of a specific schema.
+
+### 4.4 Pull replication
+
+In an imperative "pull" replication model, a target will once or periodically
+traverse the source registry contents and apply differences between the source
+state and its own state. 
+
+The pulling target might be configured to copy changes of the full registry, of
+a specific schema group, or just of a specific schema.
+
+### 4.5. Event-driven replication
+
+Event-driven replication uses CloudEvents to notify targets of changes in the
+source such that the targets can [pull](#44-pull-replication) changes
+immediately and as they become available.
+
+Each registry SHOULD offer a [Subscription API](../subscriptions-api.md)
+endpoint, either directly or using some middleware, to allow replication targets
+to subscribe to change events.
+
+The subscription's scope MUST be root of the schema registry. A [prefix
+filter](../subscriptions-api.md#3231-basic-filter-dialect) can be used to to
+scope the subscription to a particular schema group or schema.
+
+The subscribing target registers a notification sink of its choosing.
+
+```HTTP
+POST https://source.example.com/subscriptions HTTP/1.1
```

Comment:
As a result of this post, we should also add an event type `io.cloudevents.schemaregistry.subscribed.v1` to give direct feedback when the subscription is registered and the subscription does not filter out that kind of event type.

Reply:
I don't understand what you want, but what you want most likely belongs into the subscription spec, @n3wscott  :)